VidCloud9/dist/iframe.js

```diff
@@ -0,0 +1,30 @@
+var iframe = new iFrame();
+iframe.on("UpdateData", async () => {
+    var video;
+    if (document.querySelector("#myVideo > div.jw-wrapper.jw-reset > div.jw-media.jw-reset > video") !== null){
```

Comment:
You don't have to check it here as you already check it in the other line. You can just declare the variable to this and check it before sending the data.